commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt

```diff
@@ -58,6 +58,31 @@ class ResourceManager(
         }
     }
 
+    fun defaultEnrollmentLabel(
```

Comment:
Keep this method and return the enrollmentLabel. Capitalize when needed. Also, `formatWithEnrollmentLabel` could call it to return get the enrollment label and we reduce duplication.

Reply:
you are right that was my original idea but I must have gotten distracted halfway through